docs/index.md

```diff
@@ -0,0 +1,20 @@
+---
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+#layout: home
+title: Home
+---
+
+
+# {{site.data.openapi-split.info.title}}
```

Comment:
@getorymckeag all of the spec's yaml is available via `{{site.data.____}}` see: https://jekyllrb.com/docs/datafiles/